### PR TITLE
Make more Fields VV and more editable

### PR DESCRIPTION
**Pull request by DTanxxx**
Added some more VV and VVAccess.ReadWrite. However some may be inappropriate and may need to be changed back.

### CODE REVIEW

**Comment by DTanxxx** [comment]
And for the engine: [https://github.com/space-wizards/RobustToolbox/pull/1284]

File: Content.Server/GameObjects/Components/Disposal/DisposalUnitComponent.cs [hunk 1]
```diff
@@ -64,6 +65,7 @@ public class DisposalUnitComponent : SharedDisposalUnitComponent, IInteractHand,
         [ViewVariables]
         private float _pressure;
 
+        [ViewVariables]
```
Comment (exp111) [inline Content.Server/GameObjects/Components/Disposal/DisposalUnitComponent.cs:68]: This is not necessary as the Property 'Engaged' is already vv

File: Content.Server/GameObjects/Components/Doors/AirlockComponent.cs [hunk 2]
```diff
@@ -39,6 +39,7 @@ public class AirlockComponent : ServerDoorComponent, IWires
 
         private CancellationTokenSource _powerWiresPulsedTimerCancel = new CancellationTokenSource();
 
+        [ViewVariables]
```
Comment (exp111) [inline Content.Server/GameObjects/Components/Doors/AirlockComponent.cs:42]: Those also have the properties already vv, so making the field vv too is pretty useless

File: Content.Server/GameObjects/Components/Doors/AirlockComponent.cs [hunk 3]
```diff
@@ -56,6 +57,7 @@ private bool PowerWiresPulsed
             }
         }
 
+        [ViewVariables]
```
Comment (exp111) [inline Content.Server/GameObjects/Components/Doors/AirlockComponent.cs:60]: Same here

**Review by exp111 — APPROVED** [review]

**Review by DrSmugleaf — APPROVED** [review]

**Pull request merged**